docs/src/passing-in-data.rst

```diff
@@ -0,0 +1,190 @@
+
+.. code-block:: ts
+
+    const bucket_name = process.env.MYBUCKET;
+
+.. _passing_in_data_from_ssm:
+
+Getting a Value from an SSM Store Variable
+==========================================
+
+To get the value of an SSM parameter store variable,
+use code like the following,
+which uses the value of the SSM variable **my-awesome-value**.
+
+.. code-block:: ts
+		
+    const myvalue = new SSMParameterProvider(this).getString("my-awesome-value");
+
+See the :doc:`context` topic for information about the :py:class:`SSMParameterProvider <@aws-cdk/cdk.SSMParameterProvider>`.
+
+.. _passing_in_data_between_stacks:
+
+Passing in Data From Another Stack
+==================================
+
+You can pass data from one stack to another stack in the same app
+by using the **export** method in one stack and the **import** method in the other stack.
+
+The following example creates a bucket on one stack
+and passes a reference to the other stack through an interface.
+
+First create a stack with a bucket.
+The stack includes a a property we use to pass the bucket's properties to the other stack.
+Note how we use the **export** method on the bucket to get it's properties and save them
+in the stack property.
+
+.. code-block:: ts
+
+    class HelloCdkStack extends cdk.Stack {
+        // Property that defines the stack you are exporting from
+        public readonly myBucketRefProps: s3.BucketRefProps;
+
+	constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
+            super(parent, name, props);
+
+            const mybucket = new s3.Bucket(this, "MyFirstBucket");
+
+            // Save bucket's *BucketRefProps*
+            this.myBucketRefProps = mybucket.export();
+	}
+    }
+
+Next create another stack that gets a reference to the other bucket
+from the properties passed in through the constructor.
+    
+.. code-block:: ts
+
+    // The class for the other stack
+    class MyCdkStack extends cdk.Stack {
+        constructor(parent: cdk.App, name: string, props: XferBucketProps) {
+            super(parent, name);
+
+            const myOtherBucket = s3.Bucket.import(this, "MyOtherBucket", props.theBucketRefProps);
+
+	    // Do something with myOtherBucket
+        }
+    }
+
+Create an interface for the second stack's properties.
+We use this interface to pass the bucket properties between the two stacks.
+
+.. code-block:: ts
+
+    // Interface we'll use to pass the bucket's properties to another stack
+    interface XferBucketProps {
+        theBucketRefProps: s3.BucketRefProps;
+    }		
+
+Finally, connect the dots in your app.
+    
+.. code-block:: ts
+
+    const app = new cdk.App(process.argv);
+
+    const myStack = new HelloCdkStack(app, "HelloCdkStack");
+    
+    new MyCdkStack(app, "MyCdkStack", {
+        theBucketRefProps: myStack.myBucketRefProps
+    });
+
+    process.stdout.write(app.run());
+
+.. _using_cfn_template:
+
+Using an Existing |CFN| Template
```

Comment:
Does this really belong in this topic? Maybe this should be part of the cloudformation topic?

Reply:
Sure, and then I'll just add a link with some text here.